staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -246,6 +246,13 @@ func NoNamespaceKeyFunc(ctx context.Context, prefix string, name string) (string
 	return key, nil
 }
 
+// Destroy implements RESTStorage.Destroy.
+func (e *Store) Destroy() {
```

Comment:
based on the PR description, I expected a default here that closed the etcd connection.

Reply:
`e.DestroyFunc` is optional this nothing is done by default. It needs to be optional since its value depends on what `opts.Decorator` returns. `opts.Decorator` is provided through `options.RESTOptions.GetRESTOptions`. And, `options` is argument of `registry.Store.CompleteWithOptions` which is called outside of the package.

Reply:
Provide breadcrumbs here in comments so future us can find out how this gets set.

Reply:
> Provide breadcrumbs here in comments so future us can find out how this gets set.

comments here are still missing.